### ISSUE TITLE
Cannot Place Items in Yard

### ISSUE THREAD

**Issue by Narchypoo** [Uuvana-Studios/longvinter-windows-client#41]
**Describe the bug**
Getting error that item is overlapping with something and nothing is there.  Started when I was trying to place plants - would not let me, so I attempted to pick up planters - now cannot put planters back down in bare area.

**To Reproduce**
1. Go to inventory
2. Click on item to place
3. Position until turns green (but it stays red) and alert flashes repeatedly that cannot place due to overlapping.

**Expected behavior**
Item should turn green and allow to place near house.

**Screenshots**
Attached.

**Desktop (please complete the following information):**
 - OS: Windows
 - Game Version 1.0
![cannotplaceitems](https://user-images.githubusercontent.com/100515134/155883368-e8e785a1-5f8f-44d5-9228-7e802ce2f3b8.jpg)

 - Steam Version [e.g. 1.0]

**Additional context**

**Comment by Narchypoo** [comment]
Just to update, this seems pretty hit or miss - one day, it won't let me put plants in and the next, it will let me put plants in just fine, but when I try to change my planters, I can't place them.  It tried to give me the same error when I attempted to place a planter on the upper left corner of my house but after playing with it for a while, I got it to place that one.  Trying to put anything on the right side of my house is hopeless at the moment.  Not sure if that's any relevant information, but thought I'd add it, just in case it helps!  I'll keep playing with it and see if I can get it to work :)
